### ISSUE TITLE
show accuracy is deprecated

### ISSUE THREAD

**Issue by jdelange** [Vict0rSch/deep_learning#5]
Thanks, very well explained for a beginner like me.

Minor note: I get

> The show accuracy argument is deprecated, instead you should pass the accuracy metric to the model at compile time: model.compile(optimizer, loss, metrics=["accuracy"])

**Comment by jdelange** [comment]
Solved my issue as follows:

1) Change`model.compile` to:
`model.compile(loss='categorical_crossentropy', optimizer=rms, metrics=['accuracy'])`
As described [here](http://keras.io/getting-started/sequential-model-guide/)
2) Remove `show_accuracy=True` from `model.fit` and `model.evaluate`

Hope this is of use to other beginners like me.

**Comment by vict0rsch** [comment]
I updated it, thanks. Next time don't hesitate to submit a pull request, it'll be faster :)

**Comment by gontthias** [comment]
good answer
